docs/rule/no-redundant-landmark-role.md

```diff
@@ -39,11 +42,20 @@ This rule **allows** the following:
 ```
 
 ```hbs
-<footer role="contentinfo"></footer>
+<nav role="navigation"></nav>
 ```
 
+## Configuration
+
+- boolean -- if `true`, default configuration is applied
+
+- object -- containing the following property:
+  - boolean -- `checkAllElementRoles` -- if `true`, the rule checks for redundancy between any semantic HTML element with a default/implicit ARIA role and the role provided
```

Comment:
Perhaps "checking roles" is already implied by the name of the rule? What do you think about `checkAllHTMLElements` as the option name?